src/frontend/screens/Library/index.tsx

```diff
@@ -515,6 +528,55 @@ export default React.memo(function Library(): JSX.Element {
       )
     }
 
+    return library
+  }, [
+    storesFilters,
+    platformsFilters,
+    epic.library,
+    epic.username,
```

Comment:
why are these `.username` and `.user_id` dependencies added here? they were not in the original list of dependencies in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/4643/files#diff-f4c668b672834abb8870d392c22f649737bdea028101f149cc0264991f8fb8faL542-L544